Assignment 3.Rmd

```diff
@@ -136,17 +143,33 @@ What patterns do you see in the plot?
 It would be useful to determine how many people are in each cluster. We can do this easily with dplyr.
 
 ```{r}
-K7 <- 
+K7 <- dplyr::count(K4, cluster)
 ```
 
-Look at the number of people in each cluster, now repeat this process for 3 rather than 2 clusters. Which cluster grouping do you think is more informative? Write your answer below:
+Look at the number of people in each cluster, now repeat this process for 3 rather than 2 clusters. 
+Which cluster grouping do you think is more informative? Write your answer below:
 
 Once you have done this, save both of your plots to the Assignment 5 file.
 
 ##Part II
 
 Now, try to do the same for the data collected in class. Create two sets of clusters, the answers to the questions and regions where people grew up. 
 
+library(tidyverse)
+
+cluster1 <- read.csv("~/Desktop/HUDK4050/assignment3/HUDK405018-cluster.csv", header = TRUE)
+View(cluster1)
+
+cluster2 <- cluster1[3:47, 20:26]
+cluster2[cluster1 == ""] <- NA
+cluster2 <- cluster2 %>% mutate_all(funs(as.numeric(levels(.))[.]))
+cluster2 <- na.omit(cluster2)
+
+cluster2[is.na(cluster22)] <- 0
+cluster2[is.na(cluster2)] <- 99
+
+cluster2 <- as.data.frame(scale(cluster2))
+
 ##Part III
 
```

Comment:
You are on the right track. Try to finish Part II and Part III of the assignment.